---
title: "Using BigRNA"
author:
- name: Sean Davis, MD, PhD
  affiliation: Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, MD
  email: seandavi@gmail.com
package: BigRNAr
output:
  BiocStyle::html_document
abstract: |
  BigRNA is a large compendium of homogeneously processed public genomics 
  datasets and accompanying metadata collected from the source repositories.
  BigRNAr connects R to the available data and provides R-based metadata 
  search and retrieval.
vignette: |
  %\VignetteIndexEntry{Vignette Title}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

# Background

BigRNA is a large compendium of homogeneously processed public genomics 
datasets and accompanying metadata collected from the source repositories.
BigRNAr connects R to the available data and provides R-based metadata 
search and retrieval.

```{r fig.cap="Overview of BigRNA information flow and connection from R. Data are collected from public repositories like NCBI Short Read Archive (SRA) and processed using **salmon**. Metadata, including experimental protocols, sample details, and study descriptions and abstracts are simultaneously mined. The BigRNA API exposes the metadata via a GraphQL endpoint and the data as individual files for each processed sample. This package, BigRNAr, connects R to these resources. ", echo=FALSE, message=FALSE}
library(DiagrammeR)
grViz('
digraph bigrna {
  subgraph cluster_0 {
    label="SRA"
    data [label="RNA-seq"]
    meta [label="metadata"]
  }
  data -> salmon -> quantifications -> download
  subgraph cluster_1 {
    label="API"
    download [label="data\ndownload"]
    graphql [label="graphql\nmetadata"]
    meta -> graphql
  }
  subgraph cluster_2 {
    style="filled"
    label="BigRNAr"
    BigRNAConnection
  }
  BigRNAConnection -> download [label="datafile"]
  BigRNAConnection -> graphql [label="gqlQuery"]
}
      ')
```



# Getting started

```{r eval=FALSE}
BiocManager::install("seandavi/BigRNA")
```




## Create BigRNA connection object

```{r}
library(knitr)
opts_chunk$set(message=FALSE, cache=FALSE)
library(BigRNAr)
bigrna = new("BigRNAConnection")
bigrna
```

# Use Cases

## Full text search of studies

```{r}
studyFullTextSearch = '
query studyFullTextSearch (
  $match: String!=""
  $cursor: Cursor=null
) {
  allStudies(
    filter: {textsearchableIndexCol: {matches: $match}}
    after: $cursor
  ) {
    edges {
      node {
        accession
        bioproject
        gse
        abstract
        alias
        attributes
        brokerName
        centerName
        description
        identifiers
        studyType
        title
        xrefs
        status
        updated
        published
        received
        visibility
        replacedBy
        metadataByExptStudyAccession {
          nodes {
            sampAccession
            exptAccession
            sampTitle
            exptLibraryStrategy
            exptLibrarySelection
          }
          totalCount
        }
      }
    }
    pageInfo {
      hasNextPage
      endCursor
    }
    totalCount
  }
}
'

x=gqlQuery(bigrna, studyFullTextSearch, 
           variables=list(match='colon & cancer'), 
           handler=dataframe_handler)
names(x)
```

```{r}
library(DT)
print(datatable(x$edges))
```


## Getting data

```{r fileslisting}
fileListQuery = '
query getFiles($cursor: Cursor=null) {
  allBigrnaFiles(
    filter: {filename: {equalTo: "quant.genes.sf.gz"}}
    after: $cursor
  ) {
    edges {
      node {
        key
        accession
        filename
      }
    }
    pageInfo {
      hasNextPage
      endCursor
    }
  }
}
'
x=gqlQuery(bigrna, fileListQuery, 
           handler=dataframe_handler)
df = x$edges
maxN = 10000
# get about 10000 sample file metadata.
while(x$pageInfo$hasNextPage & nrow(df)<maxN)  {
  x = gqlQuery(bigrna, fileListQuery, 
          variables=list(cursor=x$pageInfo$endCursor), 
           handler=dataframe_handler)
  df = dplyr::bind_rows(df, x$edges)
}
head(df)
```


## Download



```{r}
fnames = sapply(df$node.key[1:10], function(path) {
  path = sub('^/','',path)
  datafile(bigrna, path)
})
```

```{r}
head(fnames)
```

From here, take a look at tximport or simply read as tsv. The format of the files will remain, but the details of paths, etc., is likely to change.


# Session info {.unnumbered}

```{r sessionInfo, echo=FALSE}
sessionInfo()
```
