<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Using BigRNA â€¢ BigRNAr</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/cerulean/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- sticky kit --><script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Using BigRNA">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-93043521-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-93043521-1');
</script>
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">BigRNAr</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1.4</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/getting_started.html">Using BigRNA</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/seandavi/BigRNAr">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><script src="getting_started_files/htmlwidgets-1.3/htmlwidgets.js"></script><script src="getting_started_files/viz-0.3/viz.js"></script><link href="getting_started_files/DiagrammeR-styles-0.2/styles.css" rel="stylesheet">
<script src="getting_started_files/grViz-binding-1.0.1/grViz.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>Using BigRNA</h1>
                        <h4 class="author">Sean Davis, MD, PhD</h4>
            <address class="author_afil">
      Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, MD<br><a class="author_email" href="mailto:#"></a><a href="mailto:seandavi@gmail.com">seandavi@gmail.com</a>
      </address>
                  
      
      <small class="dont-index">Source: <a href="https://github.com/seandavi/BigRNAr/blob/master/vignettes/getting_started.Rmd"><code>vignettes/getting_started.Rmd</code></a></small>
      <div class="hidden name"><code>getting_started.Rmd</code></div>

    </div>

    
        <div class="abstract">
      <p class="abstract">Abstract</p>
      BigRNA is a large compendium of homogeneously processed public genomics datasets and accompanying metadata collected from the source repositories. BigRNAr connects R to the available data and provides R-based metadata search and retrieval.
    </div>
    
<div id="background" class="section level1">
<h1 class="hasAnchor">
<a href="#background" class="anchor"></a>Background</h1>
<p>BigRNA is a large compendium of homogeneously processed public genomics datasets and accompanying metadata collected from the source repositories. BigRNAr connects R to the available data and provides R-based metadata search and retrieval.</p>
<div class="figure">
<div id="htmlwidget-64ca41e29394a5d0c80a" style="width:700px;height:432.632880098888px;" class="grViz html-widget"></div>
<script type="application/json" data-for="htmlwidget-64ca41e29394a5d0c80a">{"x":{"diagram":"\ndigraph bigrna {\n  subgraph cluster_0 {\n    label=\"SRA\"\n    data [label=\"RNA-seq\"]\n    meta [label=\"metadata\"]\n  }\n  data -> salmon -> quantifications -> download\n  subgraph cluster_1 {\n    label=\"API\"\n    download [label=\"data\ndownload\"]\n    graphql [label=\"graphql\nmetadata\"]\n    meta -> graphql\n  }\n  subgraph cluster_2 {\n    style=\"filled\"\n    label=\"BigRNAr\"\n    BigRNAConnection\n  }\n  BigRNAConnection -> download [label=\"datafile\"]\n  BigRNAConnection -> graphql [label=\"gqlQuery\"]\n}\n      ","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script><p class="caption">
Overview of BigRNA information flow and connection from R. Data are collected from public repositories like NCBI Short Read Archive (SRA) and processed using <strong>salmon</strong>. Metadata, including experimental protocols, sample details, and study descriptions and abstracts are simultaneously mined. The BigRNA API exposes the metadata via a GraphQL endpoint and the data as individual files for each processed sample. This package, BigRNAr, connects R to these resources.
</p>
</div>
</div>
<div id="getting-started" class="section level1">
<h1 class="hasAnchor">
<a href="#getting-started" class="anchor"></a>Getting started</h1>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1">BiocManager<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/BiocManager/topics/install">install</a></span>(<span class="st">"seandavi/BigRNA"</span>)</a></code></pre></div>
<div id="create-bigrna-connection-object" class="section level2">
<h2 class="hasAnchor">
<a href="#create-bigrna-connection-object" class="anchor"></a>Create BigRNA connection object</h2>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(knitr)</a>
<a class="sourceLine" id="cb2-2" data-line-number="2">opts_chunk<span class="op">$</span><span class="kw">set</span>(<span class="dt">message=</span><span class="ot">FALSE</span>, <span class="dt">cache=</span><span class="ot">FALSE</span>)</a>
<a class="sourceLine" id="cb2-3" data-line-number="3"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(BigRNAr)</a>
<a class="sourceLine" id="cb2-4" data-line-number="4">bigrna =<span class="st"> </span><span class="kw">new</span>(<span class="st">"BigRNAConnection"</span>)</a>
<a class="sourceLine" id="cb2-5" data-line-number="5">bigrna</a></code></pre></div>
<pre><code>## An object of class "BigRNAConnection"
## Slot "url":
## [1] "http://bigrna.cancerdatasci.org/"
## 
## Slot "bfc":
## class: BiocFileCache
## bfccache: /Users/sdavis2/Library/Caches/BigRNA
## bfccount: 10
## For more information see: bfcinfo() or bfcquery()</code></pre>
</div>
</div>
<div id="use-cases" class="section level1">
<h1 class="hasAnchor">
<a href="#use-cases" class="anchor"></a>Use Cases</h1>
<div id="full-text-search-of-studies" class="section level2">
<h2 class="hasAnchor">
<a href="#full-text-search-of-studies" class="anchor"></a>Full text search of studies</h2>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1">studyFullTextSearch =<span class="st"> '</span></a>
<a class="sourceLine" id="cb4-2" data-line-number="2"><span class="st">query studyFullTextSearch (</span></a>
<a class="sourceLine" id="cb4-3" data-line-number="3"><span class="st">  $match: String!=""</span></a>
<a class="sourceLine" id="cb4-4" data-line-number="4"><span class="st">  $cursor: Cursor=null</span></a>
<a class="sourceLine" id="cb4-5" data-line-number="5"><span class="st">) {</span></a>
<a class="sourceLine" id="cb4-6" data-line-number="6"><span class="st">  allStudies(</span></a>
<a class="sourceLine" id="cb4-7" data-line-number="7"><span class="st">    filter: {textsearchableIndexCol: {matches: $match}}</span></a>
<a class="sourceLine" id="cb4-8" data-line-number="8"><span class="st">    after: $cursor</span></a>
<a class="sourceLine" id="cb4-9" data-line-number="9"><span class="st">  ) {</span></a>
<a class="sourceLine" id="cb4-10" data-line-number="10"><span class="st">    edges {</span></a>
<a class="sourceLine" id="cb4-11" data-line-number="11"><span class="st">      node {</span></a>
<a class="sourceLine" id="cb4-12" data-line-number="12"><span class="st">        accession</span></a>
<a class="sourceLine" id="cb4-13" data-line-number="13"><span class="st">        bioproject</span></a>
<a class="sourceLine" id="cb4-14" data-line-number="14"><span class="st">        gse</span></a>
<a class="sourceLine" id="cb4-15" data-line-number="15"><span class="st">        abstract</span></a>
<a class="sourceLine" id="cb4-16" data-line-number="16"><span class="st">        alias</span></a>
<a class="sourceLine" id="cb4-17" data-line-number="17"><span class="st">        attributes</span></a>
<a class="sourceLine" id="cb4-18" data-line-number="18"><span class="st">        brokerName</span></a>
<a class="sourceLine" id="cb4-19" data-line-number="19"><span class="st">        centerName</span></a>
<a class="sourceLine" id="cb4-20" data-line-number="20"><span class="st">        description</span></a>
<a class="sourceLine" id="cb4-21" data-line-number="21"><span class="st">        identifiers</span></a>
<a class="sourceLine" id="cb4-22" data-line-number="22"><span class="st">        studyType</span></a>
<a class="sourceLine" id="cb4-23" data-line-number="23"><span class="st">        title</span></a>
<a class="sourceLine" id="cb4-24" data-line-number="24"><span class="st">        xrefs</span></a>
<a class="sourceLine" id="cb4-25" data-line-number="25"><span class="st">        status</span></a>
<a class="sourceLine" id="cb4-26" data-line-number="26"><span class="st">        updated</span></a>
<a class="sourceLine" id="cb4-27" data-line-number="27"><span class="st">        published</span></a>
<a class="sourceLine" id="cb4-28" data-line-number="28"><span class="st">        received</span></a>
<a class="sourceLine" id="cb4-29" data-line-number="29"><span class="st">        visibility</span></a>
<a class="sourceLine" id="cb4-30" data-line-number="30"><span class="st">        replacedBy</span></a>
<a class="sourceLine" id="cb4-31" data-line-number="31"><span class="st">        metadataByExptStudyAccession {</span></a>
<a class="sourceLine" id="cb4-32" data-line-number="32"><span class="st">          nodes {</span></a>
<a class="sourceLine" id="cb4-33" data-line-number="33"><span class="st">            sampAccession</span></a>
<a class="sourceLine" id="cb4-34" data-line-number="34"><span class="st">            exptAccession</span></a>
<a class="sourceLine" id="cb4-35" data-line-number="35"><span class="st">            sampTitle</span></a>
<a class="sourceLine" id="cb4-36" data-line-number="36"><span class="st">            exptLibraryStrategy</span></a>
<a class="sourceLine" id="cb4-37" data-line-number="37"><span class="st">            exptLibrarySelection</span></a>
<a class="sourceLine" id="cb4-38" data-line-number="38"><span class="st">          }</span></a>
<a class="sourceLine" id="cb4-39" data-line-number="39"><span class="st">          totalCount</span></a>
<a class="sourceLine" id="cb4-40" data-line-number="40"><span class="st">        }</span></a>
<a class="sourceLine" id="cb4-41" data-line-number="41"><span class="st">      }</span></a>
<a class="sourceLine" id="cb4-42" data-line-number="42"><span class="st">    }</span></a>
<a class="sourceLine" id="cb4-43" data-line-number="43"><span class="st">    pageInfo {</span></a>
<a class="sourceLine" id="cb4-44" data-line-number="44"><span class="st">      hasNextPage</span></a>
<a class="sourceLine" id="cb4-45" data-line-number="45"><span class="st">      endCursor</span></a>
<a class="sourceLine" id="cb4-46" data-line-number="46"><span class="st">    }</span></a>
<a class="sourceLine" id="cb4-47" data-line-number="47"><span class="st">    totalCount</span></a>
<a class="sourceLine" id="cb4-48" data-line-number="48"><span class="st">  }</span></a>
<a class="sourceLine" id="cb4-49" data-line-number="49"><span class="st">}</span></a>
<a class="sourceLine" id="cb4-50" data-line-number="50"><span class="st">'</span></a>
<a class="sourceLine" id="cb4-51" data-line-number="51"></a>
<a class="sourceLine" id="cb4-52" data-line-number="52">x=<span class="kw"><a href="../reference/gqlQuery.html">gqlQuery</a></span>(bigrna, studyFullTextSearch, </a>
<a class="sourceLine" id="cb4-53" data-line-number="53">           <span class="dt">variables=</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/list">list</a></span>(<span class="dt">match=</span><span class="st">'colon &amp; cancer'</span>), </a>
<a class="sourceLine" id="cb4-54" data-line-number="54">           <span class="dt">handler=</span>dataframe_handler)</a>
<a class="sourceLine" id="cb4-55" data-line-number="55"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(x)</a></code></pre></div>
<pre><code>## [1] "edges"      "pageInfo"   "totalCount"</code></pre>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(DT)</a>
<a class="sourceLine" id="cb6-2" data-line-number="2"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/print">print</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/DT/topics/datatable">datatable</a></span>(x<span class="op">$</span>edges))</a></code></pre></div>
</div>
<div id="getting-data" class="section level2">
<h2 class="hasAnchor">
<a href="#getting-data" class="anchor"></a>Getting data</h2>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb7-1" data-line-number="1">fileListQuery =<span class="st"> '</span></a>
<a class="sourceLine" id="cb7-2" data-line-number="2"><span class="st">query getFiles($cursor: Cursor=null) {</span></a>
<a class="sourceLine" id="cb7-3" data-line-number="3"><span class="st">  allBigrnaFiles(</span></a>
<a class="sourceLine" id="cb7-4" data-line-number="4"><span class="st">    filter: {filename: {equalTo: "quant.genes.sf.gz"}}</span></a>
<a class="sourceLine" id="cb7-5" data-line-number="5"><span class="st">    after: $cursor</span></a>
<a class="sourceLine" id="cb7-6" data-line-number="6"><span class="st">  ) {</span></a>
<a class="sourceLine" id="cb7-7" data-line-number="7"><span class="st">    edges {</span></a>
<a class="sourceLine" id="cb7-8" data-line-number="8"><span class="st">      node {</span></a>
<a class="sourceLine" id="cb7-9" data-line-number="9"><span class="st">        key</span></a>
<a class="sourceLine" id="cb7-10" data-line-number="10"><span class="st">        accession</span></a>
<a class="sourceLine" id="cb7-11" data-line-number="11"><span class="st">        filename</span></a>
<a class="sourceLine" id="cb7-12" data-line-number="12"><span class="st">      }</span></a>
<a class="sourceLine" id="cb7-13" data-line-number="13"><span class="st">    }</span></a>
<a class="sourceLine" id="cb7-14" data-line-number="14"><span class="st">    pageInfo {</span></a>
<a class="sourceLine" id="cb7-15" data-line-number="15"><span class="st">      hasNextPage</span></a>
<a class="sourceLine" id="cb7-16" data-line-number="16"><span class="st">      endCursor</span></a>
<a class="sourceLine" id="cb7-17" data-line-number="17"><span class="st">    }</span></a>
<a class="sourceLine" id="cb7-18" data-line-number="18"><span class="st">  }</span></a>
<a class="sourceLine" id="cb7-19" data-line-number="19"><span class="st">}</span></a>
<a class="sourceLine" id="cb7-20" data-line-number="20"><span class="st">'</span></a>
<a class="sourceLine" id="cb7-21" data-line-number="21">x=<span class="kw"><a href="../reference/gqlQuery.html">gqlQuery</a></span>(bigrna, fileListQuery, </a>
<a class="sourceLine" id="cb7-22" data-line-number="22">           <span class="dt">handler=</span>dataframe_handler)</a>
<a class="sourceLine" id="cb7-23" data-line-number="23">df =<span class="st"> </span>x<span class="op">$</span>edges</a>
<a class="sourceLine" id="cb7-24" data-line-number="24">maxN =<span class="st"> </span><span class="dv">10000</span></a>
<a class="sourceLine" id="cb7-25" data-line-number="25"><span class="co"># get about 10000 sample file metadata.</span></a>
<a class="sourceLine" id="cb7-26" data-line-number="26"><span class="cf">while</span>(x<span class="op">$</span>pageInfo<span class="op">$</span>hasNextPage <span class="op">&amp;</span><span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/nrow">nrow</a></span>(df)<span class="op">&lt;</span>maxN)  {</a>
<a class="sourceLine" id="cb7-27" data-line-number="27">  x =<span class="st"> </span><span class="kw"><a href="../reference/gqlQuery.html">gqlQuery</a></span>(bigrna, fileListQuery, </a>
<a class="sourceLine" id="cb7-28" data-line-number="28">          <span class="dt">variables=</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/list">list</a></span>(<span class="dt">cursor=</span>x<span class="op">$</span>pageInfo<span class="op">$</span>endCursor), </a>
<a class="sourceLine" id="cb7-29" data-line-number="29">           <span class="dt">handler=</span>dataframe_handler)</a>
<a class="sourceLine" id="cb7-30" data-line-number="30">  df =<span class="st"> </span>dplyr<span class="op">::</span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/bind.html">bind_rows</a></span>(df, x<span class="op">$</span>edges)</a>
<a class="sourceLine" id="cb7-31" data-line-number="31">}</a>
<a class="sourceLine" id="cb7-32" data-line-number="32"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(df)</a></code></pre></div>
<pre><code>##                                                 node.key node.accession
## 1 results/10090/gencode/M19/SRX4147707/quant.genes.sf.gz     SRX4147707
## 2 results/10090/gencode/vM19/DRX000988/quant.genes.sf.gz      DRX000988
## 3 results/10090/gencode/vM19/DRX000989/quant.genes.sf.gz      DRX000989
## 4 results/10090/gencode/vM19/DRX000990/quant.genes.sf.gz      DRX000990
## 5 results/10090/gencode/vM19/DRX000991/quant.genes.sf.gz      DRX000991
## 6 results/10090/gencode/vM19/DRX001048/quant.genes.sf.gz      DRX001048
##       node.filename
## 1 quant.genes.sf.gz
## 2 quant.genes.sf.gz
## 3 quant.genes.sf.gz
## 4 quant.genes.sf.gz
## 5 quant.genes.sf.gz
## 6 quant.genes.sf.gz</code></pre>
</div>
<div id="download" class="section level2">
<h2 class="hasAnchor">
<a href="#download" class="anchor"></a>Download</h2>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb9-1" data-line-number="1">fnames =<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/lapply">sapply</a></span>(df<span class="op">$</span>node.key[<span class="dv">1</span><span class="op">:</span><span class="dv">10</span>], <span class="cf">function</span>(path) {</a>
<a class="sourceLine" id="cb9-2" data-line-number="2">  path =<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/grep">sub</a></span>(<span class="st">'^/'</span>,<span class="st">''</span>,path)</a>
<a class="sourceLine" id="cb9-3" data-line-number="3">  <span class="kw"><a href="../reference/datafile.html">datafile</a></span>(bigrna, path)</a>
<a class="sourceLine" id="cb9-4" data-line-number="4">})</a></code></pre></div>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb10-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(fnames)</a></code></pre></div>
<pre><code>##                results/10090/gencode/M19/SRX4147707/quant.genes.sf.gz 
## "/Users/sdavis2/Library/Caches/BigRNA/89aa1bd5f503_quant.genes.sf.gz" 
##                results/10090/gencode/vM19/DRX000988/quant.genes.sf.gz 
## "/Users/sdavis2/Library/Caches/BigRNA/89aa7bc8a63b_quant.genes.sf.gz" 
##                results/10090/gencode/vM19/DRX000989/quant.genes.sf.gz 
## "/Users/sdavis2/Library/Caches/BigRNA/89aa3219aafa_quant.genes.sf.gz" 
##                results/10090/gencode/vM19/DRX000990/quant.genes.sf.gz 
## "/Users/sdavis2/Library/Caches/BigRNA/89aa33281cc8_quant.genes.sf.gz" 
##                results/10090/gencode/vM19/DRX000991/quant.genes.sf.gz 
##  "/Users/sdavis2/Library/Caches/BigRNA/89aae79a8b5_quant.genes.sf.gz" 
##                results/10090/gencode/vM19/DRX001048/quant.genes.sf.gz 
## "/Users/sdavis2/Library/Caches/BigRNA/89aa55330a7f_quant.genes.sf.gz"</code></pre>
<p>From here, take a look at tximport or simply read as tsv. The format of the files will remain, but the details of paths, etc., is likely to change.</p>
</div>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1 class="hasAnchor">
<a href="#session-info" class="anchor"></a>Session info</h1>
<pre><code>## R Under development (unstable) (2019-01-14 r75992)
## Platform: x86_64-apple-darwin15.6.0 (64-bit)
## Running under: macOS Mojave 10.14.2
## 
## Matrix products: default
## BLAS: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
## LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
## [1] DT_0.5           BigRNAr_0.1.3    knitr_1.22       DiagrammeR_1.0.1
## [5] BiocStyle_2.11.0
## 
## loaded via a namespace (and not attached):
##  [1] viridis_0.5.1       httr_1.4.0          tidyr_0.8.3        
##  [4] bit64_0.9-7         jsonlite_1.6        viridisLite_0.3.0  
##  [7] shiny_1.3.0         assertthat_0.2.1    BiocManager_1.30.4 
## [10] BiocFileCache_1.7.7 blob_1.1.1          yaml_2.2.0         
## [13] pillar_1.3.1        RSQLite_2.1.1       backports_1.1.3    
## [16] glue_1.3.1          downloader_0.4      digest_0.6.18      
## [19] RColorBrewer_1.1-2  promises_1.0.1      colorspace_1.4-1   
## [22] htmltools_0.3.6     httpuv_1.5.1        plyr_1.8.4         
## [25] XML_3.98-1.19       pkgconfig_2.0.2     bookdown_0.9       
## [28] xtable_1.8-3        purrr_0.3.2         scales_1.0.0       
## [31] brew_1.0-6          later_0.8.0         tibble_2.1.1       
## [34] ggplot2_3.1.1       influenceR_0.1.0    lazyeval_0.2.2     
## [37] rgexf_0.15.3        mime_0.6            magrittr_1.5       
## [40] crayon_1.3.4        memoise_1.1.0       evaluate_0.13      
## [43] fs_1.2.7            MASS_7.3-51.4       xml2_1.2.0         
## [46] Rook_1.1-1          tools_3.6.0         hms_0.4.2          
## [49] stringr_1.4.0       munsell_0.5.0       compiler_3.6.0     
## [52] pkgdown_1.3.0       rlang_0.3.4         grid_3.6.0         
## [55] rstudioapi_0.10     rappdirs_0.3.1      htmlwidgets_1.3    
## [58] visNetwork_2.0.6    crosstalk_1.0.0     igraph_1.2.4       
## [61] rmarkdown_1.12      gtable_0.3.0        DBI_1.0.0          
## [64] roxygen2_6.1.1      curl_3.3            R6_2.4.0           
## [67] gridExtra_2.3       dplyr_0.8.0.1       bit_1.1-14         
## [70] commonmark_1.7      rprojroot_1.3-2     readr_1.3.1        
## [73] desc_1.2.0          stringi_1.4.3       Rcpp_1.0.1         
## [76] dbplyr_1.3.0        tidyselect_0.2.5    xfun_0.6</code></pre>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#background">Background</a></li>
      <li>
<a href="#getting-started">Getting started</a><ul class="nav nav-pills nav-stacked">
<li><a href="#create-bigrna-connection-object">Create BigRNA connection object</a></li>
      </ul>
</li>
      <li>
<a href="#use-cases">Use Cases</a><ul class="nav nav-pills nav-stacked">
<li><a href="#full-text-search-of-studies">Full text search of studies</a></li>
      <li><a href="#getting-data">Getting data</a></li>
      <li><a href="#download">Download</a></li>
      </ul>
</li>
      <li><a href="#session-info">Session info</a></li>
      </ul>
</div>
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by Sean Davis.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
</div>

  

  </body>
</html>
